workshop/library/ui/components/toolTip.lua

```diff
@@ -28,7 +28,11 @@ return {
 
         if orientation == "vertical" then -- If orientation is specified to "vertical"
             local container = teverse.construct("guiFrame", {
+<<<<<<< HEAD
```

Comment:
bad merge here too?

Reply:
Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.